### PR TITLE
Initial app changes with readme

### PR DESCRIPTION
**Pull request by jeffbanks**
Initial additions for Netflix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,106 @@
...
-
-CREATE TABLE demo.users (
-    lastname text PRIMARY KEY,
-    age int,
-    city text,
-    email text,
-    firstname text);
-```
+* [app.py](app.py) - main application file
+* [netflix-shows.cql](netflix-shows.cql) - Use this file to create the schema
```
Comment (jgillenwater) [inline README.md]:
```suggestion
* [netflix-shows.cql](netflix-shows.cql) - file to create the schema
```

File: README.md [hunk 2]
```diff
@@ -1,100 +1,106 @@
...
 
-## Connect to your cluster
+## How this works
+To get started, read through the comments in the `app.py` to familiarize yourself with the steps for interacting with your own Cassandra database. The functions invoked by the `app.py` are created to provide more flexibility for modifications as you learn.
```
Comment (jgillenwater) [inline README.md]:
```suggestion
To get started, read the `app.py` comments to learn the steps for interacting with your Cassandra database. The functions invoked by the `app.py` are created to provide more flexibility for modifications as you learn.
```

File: README.md [hunk 3]
```diff
@@ -1,100 +1,106 @@
...
+## Setup and running
 
-```python
-def create_connection():
-    # TO DO: Fill in your own contact point
-    cluster = Cluster(['127.0.0.1'])
-    return cluster.connect('demo')
-```
+### Prerequisites
+If using [DataStax Desktop](https://www.datastax.com/blog/2020/05/learn-cassandra-datastax-desktop), no prerequisites will be required. The Cassandra instance is provided with the DataStax Desktop stack as part of container provisioning.
```
Comment (jgillenwater) [inline README.md]:
```suggestion
If using [DataStax Desktop](https://www.datastax.com/blog/2020/05/learn-cassandra-datastax-desktop), no prerequisites are required. The Cassandra instance is provided with the DataStax Desktop stack as part of container provisioning.
```

File: README.md [hunk 4]
```diff
@@ -1,100 +1,106 @@
...
 
-## CRUD Operations
-Fill the code in the functions that will add a user, get a user, update a user and delete a user from the table with the driver.
+If NOT using DataStax Desktop, spin up your own local instance of Cassandra exposing its address and port to align with the settings in the **app.py** file.  You will need to install and perform the following steps:
```
Comment (jgillenwater) [inline README.md]:
```suggestion
If NOT using DataStax Desktop, spin up your own local instance of Cassandra exposing its address and port to align with the settings in the `app.py` file.  You will need to install and perform the following steps:
```

File: README.md [hunk 5]
```diff
@@ -1,100 +1,106 @@
...
-def get_user(session, lastname):
-    # TO DO: execute SimpleStatement that retrieves one user from the table
-    # TO DO: print firstname and age of user
-    result = session.execute("SELECT * FROM users WHERE lastname = %s", [lastname]).one()
-    print result.firstname, result.age
-```
+* A running instance of Apache Cassandra® 2.1+
+* Python 2.7, 3.4, 3.5, or 3.6.
+* Use Pip to install the driver: pip install cassandra-driver
+* We highly recommend to use a virtualenv
```
Comment (jgillenwater) [inline README.md]:
```suggestion
* Installed Cassandra driver: `pip install cassandra-driver`
* virtualenv (recommended)
```

File: README.md [hunk 6]
```diff
@@ -1,100 +1,106 @@
...
+});
 ```
- ## License
-Copyright 2019 Rebecca Mills
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+## Running
+Simply kickoff the app from the command line.
```
Comment (jgillenwater) [inline README.md]:
```suggestion
Start the app from the command line.
```

**Review by jgillenwater — COMMENTED** [review]
Minor changes to be made.

**Comment by jeffbanks** [comment]
Thanks @jgillenwater I liked the suggested changes!  I'll add a ticket to get those applied to the other language experiences so we have consistency where applicable.

**Comment by jgillenwater** [comment]
Great! I didn't check the code. Is it safe to assume it has all been tested?

**Comment by jeffbanks** [comment]
> Great! I didn't check the code. Is it safe to assume it has all been tested?

That is correct, local testing of Python 2.x has occurred and working as expected.  We have a ticket for overall OS and version tests for additional test coverage.

**Review by csplinter — APPROVED** [review]

**Pull request merged**